src/views/GovernanceReviewTeam/Actions/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Button } from '@trussworks/react-uswds';
+
+import { RadioField, RadioGroup } from 'components/shared/RadioField';
+import { BusinessCaseModel } from 'types/businessCase';
+
+type ActionsProps = {
+  businessCase: BusinessCaseModel;
+};
+
+const Actions = ({ businessCase }: ActionsProps) => {
+  const history = useHistory();
+
+  const businessCaseExists = !!businessCase.id;
+  const [actionRoute, setActionRoute] = useState('');
+
+  const onChange = (e: React.ChangeEvent<HTMLInputElement>) =>
+    setActionRoute(e.target.value);
+
+  const onSubmit = () => history.push(`actions/${actionRoute}`);
+
+  const NotAnITRequest = (
+    <RadioField
+      key="not-an-it-request"
+      id="not-an-it-request"
+      inline={false}
```

Comment:
Argh, `RadioField` should probably have the `inline` prop as optional, so we don't need to specify it. Doesn't need to be accomplished in this PR, but it's a small fix we should make at some point.